### PR TITLE
Add language types to the code blocks :fox_face:

### PR DESCRIPTION
**Pull request by blakedrumm**

### CODE REVIEW

**Comment by blakedrumm** [comment]
#sign-off

**Comment by Jak-MS** [comment]
@MashaMSFT 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by blakedrumm** [comment]
Any update on this? @MashaMSFT

File: docs/database-engine/availability-groups/windows/create-or-configure-an-availability-group-listener-sql-server.md [hunk 1]
```diff
@@ -156,7 +156,7 @@ helpviewer_keywords:
   
      For example, the following **New-SqlAvailabilityGroupListener** command creates an availability group listener named `MyListener` for the availability group `MyAg`. This listener will use the IPv4 address passed to the **-StaticIp** parameter as its virtual IP address.  
   
-    ```  
+    ```sql
```
Comment (MashaMSFT) [inline docs/database-engine/availability-groups/windows/create-or-configure-an-availability-group-listener-sql-server.md]:
```suggestion
    ```powershell  
```

File: docs/database-engine/availability-groups/windows/create-or-configure-an-availability-group-listener-sql-server.md [hunk 2]
```diff
@@ -168,7 +168,7 @@ helpviewer_keywords:
   
      For example, the following **Set-SqlAvailabilityGroupListener** command sets the port number for the availability group listener named `MyListener` to `1535`. This port is used to listen for connections to the listener.  
   
-    ```  
+    ```sql
```
Comment (MashaMSFT) [inline docs/database-engine/availability-groups/windows/create-or-configure-an-availability-group-listener-sql-server.md]:
```suggestion
    ```powershell  
```

File: docs/database-engine/availability-groups/windows/create-or-configure-an-availability-group-listener-sql-server.md [hunk 3]
```diff
@@ -179,7 +179,7 @@ helpviewer_keywords:
   
      For example, the following **Add-SqlAGListenerstaticIp** command adds a static IPv4 address to the availability group listener `MyListener` on the availability group `MyAg`. This IPv6 address serves as the virtual IP address of the listener on the subnet `255.255.252.0`. If the availability group spans multiple subnets, you should add a static IP address for each subnet to the listener.  
   
-    ```  
+    ```sql
```
Comment (MashaMSFT) [inline docs/database-engine/availability-groups/windows/create-or-configure-an-availability-group-listener-sql-server.md]:
```suggestion
    ```powershell
```

**Review by MashaMSFT — CHANGES_REQUESTED** [review]
these are powershell commands, not t-sql

**Comment by MashaMSFT** [comment]
Hi @blakedrumm , thanks for the ping! My apologies for the delay. These code blocks are PowerShell commands, not Transact-SQL, so I've submitted some changes. 

Thanks!
Masha from the SQL Docs team

**Comment by MashaMSFT** [comment]
#sign-off

**Comment by blakedrumm** [comment]
@MashaMSFT - I would have pinged you directly on Microsoft Teams but I could not find your Teams alias. If you could update your GitHub profile with your full name so you can be found inside Microsoft Directory it would have sped things up for this pull request.

Just a suggestion though, glad to have this signed-off. And my apologies for the code block in TSQL that were PowerShell. Thank you!! :smiley:

**Pull request merged**